components/breadcrumb/data-binding.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Data Binding
+page_title: Breadcrumb - Data Binding
+description: Data Binding basics in the Breadcrumb for Blazor.
+slug: breadcrumb-data-binding
+tags: telerik,blazor,breadcrumb,data,binding
+published: True
+position: 5
+---
+
+# Breadcrumb Data Binding
+
+This article explains how to provide data to a Breadcrumb component, so it renders items for you, the properties related to data binding and their results.
```

Comment:
The sentence contains a subordinate clause ("so it renders items for you"), which breaks its flow of listed topics. We should either use a bullet list (which will repeat the list below), split the sentence to two, or make it simpler by removing "so it renders items for you".

Reply:
Simplified the sentence.